src/test/java/io/split/android/engine/impressions/ImpressionsStorageTest.java

```diff
@@ -0,0 +1,194 @@
+package io.split.android.engine.impressions;
```

Comment:
Can you have tests that if it failes 3 attemptsr make sure it is deleted, or if the impression is too old also deleted?

Reply:
Added test to check max attempts limit reached. 
About outdated impressions, the getStoredImpressions test checks that, because one of the initial impression is outdated.